src/protobuf.cpp

```diff
@@ -34,18 +34,22 @@ void ProtoBuffer::ensureCapacity(size_t new_data_size) {
     _data = (unsigned char*) realloc(_data, _capacity);
 }
 
-void ProtoBuffer::putVarInt(u64 n) {
-    _offset = putVarInt(_offset, n);
+void ProtoBuffer::putVarInt(u32 n) {
```

Comment:
Why did you add an overload for `u32` when it's implementation is exactly the same as for `u64`?

Reply:
The idea was to save some stack memory by not having a `u64` when a `u32` is enough, if it's not worth the extra effort I'll remove it

Reply:
93b4189c57119791275cf6bebb17571be243e911

Reply:
The function does not take stack memory, all computation is in registers.